libcperciva/network/network_connect.c

```diff
@@ -15,7 +15,7 @@ struct connect_cookie {
 	int (* callback)(void *, int);
 	void * cookie;
 	struct sock_addr * const * sas;
-	struct sock_addr * sa_b;
+	const struct sock_addr * sa_b;
```

Comment:
This resolves a compiler warning, but I haven't thought about what it's actually doing.  I'm not at all certain that this is the correct fix.

Reply:
Here's the relevant error (without that commit):

```
../libcperciva/network/network_connect.c:227:10: error: assigning to 'struct sock_addr *' from 'const struct sock_addr *' discards qualifiers [-Werror,-Wincompatible-pointer-types-discards-qualifiers]
        C->sa_b = sa_b;
                ^ ~~~~
```